liiv2021a/openreview/packages.tex

```diff
@@ -0,0 +1,19 @@
+\usepackage{neurips_2019}
```

Comment:
Please remove the neurips_2019 package, as we are using ReScience styles. The compilation (with default make) fails due to this import.

Reply:
Looks like we pushed the wrong version of our packages.tex, it should be fixed now.